### PR TITLE
feat!: rename variables

### PR DESCRIPTION
**Pull request by TomTaehoonKim**
# Changes

Rename 'validator reward ratio' to 'validator reward scalar' since it is improper to call a numerator as ratio.

### CODE REVIEW

**Comment by chokobole** [comment]
The commit type should be `feat!` and may add `BREAKING CHANGE` to notify users that the existing sdks may not work.

File: packages/contracts/contracts/L2/L1Block.sol [hunk 1]
```diff
@@ -60,9 +60,9 @@ contract L1Block is Semver {
     uint256 public l1FeeScalar;
 
     /**
-     * @notice The ratio to distribute transaction fees as validator reward. 4 decimal.
+     * @notice The scalar to distribute transaction fees as validator reward. The denominator is 10000, so the ratio is expressed in 4 decimal places.
```
Comment (seolaoh) [inline packages/contracts/contracts/L2/L1Block.sol:63]: Max line length of a solidity file is forced to 100. See the [solhint setting](https://github.com/kroma-network/kroma/blob/dev/packages/contracts/.solhint.json#L8).

**Comment by seolaoh** [comment]
You'd better to match the commit message with the convention. [I think `!feat` should be `feat!`](https://github.com/kroma-network/kroma/blob/dev/.gitmessage#L43). Also, [the commit message should not exceed max 80 characters in one line](https://github.com/kroma-network/kroma/blob/dev/.gitmessage#L75).

File: packages/contracts/contracts/L2/L1Block.sol (hunk 1 above)
Comment (TomTaehoonKim) [reply packages/contracts/contracts/L2/L1Block.sol]: Applied.

File: specs/exec-engine.md [hunk 2]
```diff
@@ -75,10 +75,10 @@ Transaction fees in Kroma are different from [eip-1559] specification.
 The Base Fee is not burned, and there is no distinction between Base Fee and Priority Fee.
 The transaction fee is distributed to two vaults, Validator Reward Vault and Protocol Fee.
 
-- Validator Reward Vault: `(baseFee + priorityFee) * ValidatorRewardRatio)`
-- Protocol Vault: `(baseFee + priorityFee) * (1 - ValidatorRewardRatio)`
+- Validator Reward Vault: `(baseFee + priorityFee) * ValidatorRewardScalar / 10000)`
```
[the remark this answers is not in the document]
Comment (TomTaehoonKim) [reply specs/exec-engine.md]: Applied.

**Comment by TomTaehoonKim** [comment]
@seolaoh Applied both changes; the commit title and body.

**Review by seolaoh — APPROVED** [review]
LGTM, but due to the ongoing QA of challenge, we are currently restricting merging of other changes not related to challenge. So plz wait until the QA is complete.

File: packages/contracts/contracts/L2/L1Block.sol [hunk 3]
```diff
@@ -60,9 +60,10 @@ contract L1Block is Semver {
     uint256 public l1FeeScalar;
 
     /**
-     * @notice The ratio to distribute transaction fees as validator reward. 4 decimal.
+     * @notice The scalar to distribute transaction fees as validator reward.
```
Comment (chokobole) [inline packages/contracts/contracts/L2/L1Block.sol:63]: To be consistent with the comment above,

```suggestion
     * @notice The scalar value to distribute transaction fees as validator reward.
```

Comment (chokobole) [reply packages/contracts/contracts/L2/L1Block.sol:63]: Plus, for the same reason, could you specify the denominator of `l1FeeScalar` above?

Comment (TomTaehoonKim) [reply packages/contracts/contracts/L2/L1Block.sol]: Both applied.

**Review by chokobole — APPROVED** [review]
LGTM

**Review by 0xHansLee — COMMENTED** [review]
Looks good to me, too.
But could you update the geth after merging https://github.com/kroma-network/go-ethereum/pull/46#pullrequestreview-1508155320?

**Comment by chokobole** [comment]
@TomTaehoonKim Please update the geth~

**Comment by TomTaehoonKim** [comment]
@chokobole @0xHansLee Sorry for the late update. Just updated!

**Comment by 0xHansLee** [comment]
@TomTaehoonKim 
Thank you for updating. However, we need to resolve a CI issue. Let's merge after resolving it.
devops team will handle it soon.

**Comment by TomTaehoonKim** [comment]
@chokobole Do we need to update kroma_l1_block.rs in zkevm-circuits after this PR is merged?

**Comment by chokobole** [comment]
> @chokobole Do we need to update kroma_l1_block.rs in zkevm-circuits after this PR is merged?

I don't think there's much change at the bytecode level since we're just renaming it.

**Comment by chokobole** [comment]
The type of the last commit should be `build`. Please see https://github.com/kroma-network/.github/blob/main/CONTRIBUTING.md#commit-type

**Review by chokobole — CHANGES_REQUESTED** [review]
The type of the last commit should be `build`. Please see https://github.com/kroma-network/.github/blob/main/CONTRIBUTING.md#commit-type

**Comment by TomTaehoonKim** [comment]
> The type of the last commit should be `build`. Please see https://github.com/kroma-network/.github/blob/main/CONTRIBUTING.md#commit-type

@chokobole Fixed.

**Review by chokobole — APPROVED** [review]
LGTM

**Review by Pangssu — APPROVED** [review]
LGTM

**Comment by TomTaehoonKim** [comment]
@Pangssu Done!

**Pull request merged**